packages/radio-player/test/radio-player.test.js

```diff
@@ -166,6 +169,78 @@ describe('Radio Player', () => {
     expect(el.searchTerm).to.equal('foo');
   });
 
+  class MockSearchHandler {
```

Comment:
is this the only place you use this mock? if not, perhaps we move it out of the `describe` block

Reply:
Yeah I'm only using it in here, but this actually got me thinking about this massive test file and I organized better so this `MockSearchHandler` is now inside a more isolated `describe('Searching')` block.